lib/ansible/modules/cloud/amazon/aws_acm_export_certificates.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/python
+# Copyright: Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: aws_acm_export_certificates
+short_description: Exports a private certificate issued by a private certificate authority (CA) for use anywhere
+description:
+    - Exports a private certificate issued by a private certificate authority (CA) for use anywhere
+version_added: "2.9"
+author:
+  - diodonfrost (@diodonfrost)
+options:
+  certificate_arn:
+    description:
+      - The Amazon Resource Name (ARN) of the issued certificate.
+    type: str
+    required: true
+  passphrase:
+    description:
+      - Passphrase to associate with the encrypted exported private key.
+    type: str
+    required: true
+extends_documentation_fragment:
+    - aws
+    - ec2
+requirements:
+    - boto3
+    - botocore
+    - python >= 2.6
+'''
+
+EXAMPLES = '''
+# Note: These examples do not set authentication details, see the AWS Guide for details.
+# Exporting a private certificate and private key (more details: https://docs.aws.amazon.com/acm/latest/userguide/gs-acm-export-private.html#export-cli)
+
+- name: obtain specific privates certificates
+  aws_acm_export_certificates:
+    certificate_arn: "arn:aws:acm:us-east-1:123456789012:certificate/12345678-1234-1234-1234-123456789012"
+    passphrase: secretpassword
+  register: acm_certificates
+'''
+
+RETURN = '''
+export:
```

Comment:
As suggested below, maybe make `certificate`, `certificate_chain` and `private_key` top level return values

Reply:
```
RETURN = '''
-export:
-    description: Export the certificate, the certificate chain, and the encrypted private key associated with the public key embedded in the certificate
+certificate:
+    Description: The base64 PEM-encoded certificate
     returned: always
-    type: dict
-    sample:
-      certificate: XXXXXXXXXXXXXXXXXXXX
-      certificate_chain: XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
-      private_key: XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+    type: string
+certificate_chain:
+    Description: The base64 PEM-encoded certificate chain. This does not include the certificate that you are exporting
+    returned: always
+    type: string
+private_key:
+    Description: The encrypted private key associated with the public key in the certificate. The key is output in PKCS #8 format and is base64 PEM-encoded
+    returned: always
+    type: string
```

Reply:
return `certificate`, `certificate_chain` and `private_key`